### PR TITLE
Improve code quality

### PR DESCRIPTION
**Pull request by junjunjieOng**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/GenerateEmailsCommand.java [hunk 1]
```diff
@@ -23,15 +23,18 @@
             + "INDEX\n"
             + "Example: " + COMMAND_WORD + " 1";
 
-    static final String MESSAGE_NO_CONTACTS_ASSIGNED =
+    public static final String MESSAGE_NO_CONTACTS_ASSIGNED =
```
Comment (snss231) [inline src/main/java/seedu/address/logic/commands/GenerateEmailsCommand.java:26]: intentionally used default access modifier as it is not used outside the package.

File: src/main/java/seedu/address/logic/parser/ImportCommandParser.java [hunk 2]
```diff
@@ -69,9 +70,9 @@ public ImportCommand parse(String args) throws ParseException {
             throw new ParseException(String.format(MESSAGE_FOLDER_SPECIFIED, file.getPath()));
         }
 
-        List<Person> toAdd = new ArrayList<>();
+        List<Person> personToAddList = new ArrayList<>();
 
-        List<String> invalidFields = new ArrayList<>();
+        List<String> invalidFieldsList = new ArrayList<>();
```
Comment (snss231) [inline src/main/java/seedu/address/logic/parser/ImportCommandParser.java:75]: why need to add the word list?

File: src/main/java/seedu/address/logic/commands/GenerateEmailsCommand.java (hunk 1 above)
Comment (junjunjieOng) [reply src/main/java/seedu/address/logic/commands/GenerateEmailsCommand.java:26]: I see i will revert it back then

File: src/main/java/seedu/address/model/task/Link.java [hunk 3]
```diff
@@ -6,6 +6,9 @@
 import java.net.URISyntaxException;
 import java.net.URL;
 
+/**
+ * Represents a link associate with the task.
```
Comment (snss231) [inline src/main/java/seedu/address/model/task/Link.java:10]: associated

File: src/main/java/seedu/address/logic/parser/ImportCommandParser.java (hunk 2 above)
Comment (junjunjieOng) [reply src/main/java/seedu/address/logic/parser/ImportCommandParser.java:75]: Was trying to match with personToAddList but i think the original is fine. Will revert back

**Review by snss231 — COMMENTED** [review]
Left some concerns, thanks for looking through

File: src/main/java/seedu/address/logic/commands/AssignCommand.java [hunk 4]
```diff
@@ -54,18 +58,18 @@ public AssignCommand(Index taskIndex, Index personIndex) {
     @Override
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
-        List<Person> shownPersonList = model.getFilteredPersonList();
-        List<Task> shownTaskList = model.getFilteredTaskList();
+        List<Person> filteredPersonList = model.getFilteredPersonList();
```
Comment (snss231) [inline src/main/java/seedu/address/logic/commands/AssignCommand.java:61]: If you are concerned about the naming, you may want to look at `EditCommand` which uses the naming `lastShownList` (I don't think it matters, should be fine either way)

**Review by brian16600 — COMMENTED** [review]
Thanks for cleaning up. LGTM, go ahead and merge

**Pull request merged**